web/themes/custom/pineapple/sass/components/paragraphs/_content-spotlight.scss

```diff
@@ -0,0 +1,23 @@
+.spotlight-container {
+  display: flex;
+  flex-wrap: wrap;
+
+  .spotlight-item {
+    display: block;
+    margin-bottom: $unit * 3;
+    position: relative;
+    width: 100%;
+    padding-right: 15px;
+    padding-left: 15px;
+
+    @include breakpoint($lap) {
+      flex: 0 0 50%;
+      max-width: 50%;
+    }
+
+    @include breakpoint($desk) {
+      flex: 0 0 33.33333%;
+      max-width: 33.33333%;
+    }
+  }
+}
```

Comment:
This code is identical to the code in the `_view--recipes.scss` file and is duplicate. create i identical class for both components so you can reuse it.

Reply:
done